docs/source/guide/pt-1-intro.md

```diff
@@ -50,56 +52,84 @@ the circuit on a noisy simulator, and returns the probability of the ground
 state. See the [Executors](executors.md) section for more information on
 how to define more advanced executors.
 
+During execution by the simulator, a coherent error is introduced in the circuit
+by applying a rotation around the X-axis (Rx gate) to each output of any 2-qubit gate.
```

Comment:
```suggestion
by applying a rotation around the X-axis (Rx gate) to each output of any 2-qubit gate in the circuit of interest.
```

Reply:
ok, but then I am removing the word circuit from the line above